src/System.IO.Compression.Brotli/System/IO/Compression/Brotli.cs

```diff
@@ -117,6 +117,10 @@ internal static void EnsureInitialized(ref State state, bool compress)
 
         public static int GetMaximumCompressedSize(int inputSize)
         {
+            if (inputSize < 0 || inputSize > 2147483132) // 2^32 - 1 - 515 (max compressed extra bytes)
```

Comment:
the result will be wrong, because in fact true number is bigger than int. We need this check because of 2 reasons:
1. We return int, so result should be less than int
2. Span size can't be more than MaxInt and we expected that after calling this method user will create span or array with result size;